utils/osdetail/binmeta_windows.go

```diff
@@ -0,0 +1,96 @@
+package osdetail
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"strings"
+)
+
+const powershellGetFileDescription = `Get-ItemProperty "%s" | Format-List`
+
```

Comment:
Should we use `%q` here instead to prevent PS code-injection using file-paths (if that's even possible)